ibm/mas_devops/roles/suite_db2_setup_for_manage/tasks/main.yml

```diff
@@ -74,6 +74,29 @@
   delay: 30 # seconds
   until: copy_result.rc == 0
 
+- name: Execute db2 restart database
+  shell: |
+    oc exec -n {{db2_namespace}} {{db2_pod_name}} -- su - db2inst1 -lc "db2stop"
+    sleep 180
+    oc exec -n {{db2_namespace}} {{db2_pod_name}} -- su - db2inst1 -lc "db2start"
+    sleep 180
+    oc exec -n {{db2_namespace}} {{db2_pod_name}} -- su - db2inst1 -lc "db2 restart database {{ db2_dbname }}"
+    sleep 180
+    oc exec -n {{db2_namespace}} {{db2_pod_name}} -- su - db2inst1 -lc "db2 activate database {{ db2_dbname }}"
+    sleep 180
+  register: db2_restart_database_output
+  ignore_errors: true
+
+- name: "Database db2 restart debug information"
+  debug:
+    msg:
+      - "Result (stdout) ........................ {{ db2_restart_database_output.stdout }}"
+      - "result (stderr) ........................ {{ db2_restart_database_output.stderr }}"
+
+- name: "Pause for 3 minutes ..."
```

Comment:
Why do we need to pause for 3 minutes?

Reply:
@whitfiea I need to retest, I believe we can remove this 3 minutes pause